src/components/Hearing.js

```diff
@@ -104,6 +113,18 @@ export class Hearing extends React.Component {
     );
   }
 
+  getManageButton() {
```

Comment:
Yep. What I'm thinking, should the toolbar actually be visible whenever we are viewing an editable hearing? Currently, clicking "Muokkaa" is required before the user can see the toolbar - is this a good precaution?